stream_alert_cli/athena/handler.py

```diff
@@ -94,6 +94,13 @@ def rebuild_partitions(athena_client, options, config):
         new_partitions_statement = athena_helpers.partition_statement(
             unique_partitions, options.bucket, options.table_name)
 
+        # Make sure our new alter table statement is within the query API limits
+        if len(new_partitions_statement) > MAX_QUERY_LENGTH:
+            LOGGER_CLI.error('Partition statement too large, writing to local file')
+            with open('partitions_{}.txt'.format(options.table_name), 'w') as partition_file:
```

Comment:
Do you think about log the `new_partitions_statement` to CloudWatch, instead of writing to a local file?

Reply:
this doesn't run in Lambda

Reply:
Oh, I see. How about logging to LOGGER_CLI?